### PR TITLE
[Weblate] Remove empty/unchanged translation strings. Add Bulgarian language. Add general translation updates. 

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary
Remove empty/unchanged translation strings. Add Bulgarian language. Add general translation updates. 

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

### CODE REVIEW

File: i18n/i18n.jsx [hunk 1]
```diff
@@ -24,6 +25,12 @@ import store from 'stores/redux_store.jsx';
 
 // should match the values in model/config.go
 const languages = {
+    bg: {
+        value: 'bg',
+        name: 'България',
+        order: 0,
```
Comment (enahum) [inline i18n/i18n.jsx:31]: please update the order of the other languages

**Comment by enahum** [comment]
This one needs a QA review

**Comment by metanerd** [comment]
> 1 - Bulgarian language addition looks good, except please also add test for it at: https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/__snapshots__/schema_admin_settings.test.jsx.snap
> 
> 2 - Question: what does "general translation updates" refer to?
> 
> 3 - Two other comments below

1. Ah nice, will do!
2. These are the general updates from this PR: https://github.com/mattermost/mattermost-webapp/pull/6347 (I just cannot modify that PR. At least, I don't know how)
3. Thank you, will fix and enable sorting on weblate side.

**Review by nevyangelova — APPROVED** [review]
Translations look good although I will say they are quite formal and I thought the language we are keeping is more personal? There are some word shortenings that don't seem necessary IMO. But I won't block the PR just curious :) 

Also there's a test failing but I think you only need to update the snapshots.

File: i18n/de.json [hunk 2]
```diff
@@ -3868,5 +3864,33 @@
   "admin.channel_list.archived": "Archiviert",
   "admin.jobTable.headerFiles": "Dateien",
   "admin.jobTable.downloadLink": "Herunterladen",
-  "warn_metric_ack_modal.learn_more.link": "Mehr erfahren"
+  "warn_metric_ack_modal.learn_more.link": "Mehr erfahren",
+  "team.button.name_undefined": "Dieses Team hat keinen Namen",
+  "team_import_tab.importHelpExportInstructions": "Slack > Administration > Workspace settings > Import/Export Data > Export > Start Export",
+  "channel_modal.displayNameError": "Anzeigename muss mindestens 2 Zeichen lang sein.",
```
[the remark this answers is not in the document]
Comment (jasonblais) [reply i18n/de.json:3870]: @metanerd These strings are still not in alphabetical order - will they be resolved in another PR? If so, I'm fine with that.

**Comment by metanerd** [comment]
@jasonblais yes, upcoming PRs will fix the alphabetical order.

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
@jgilliam17 this is ready for QA review

**Review by jgilliam17 — CHANGES_REQUESTED** [review]
Thank you @metanerd 
- Verified Bulgarian language has been added in Account Settings>Display>Language 
I noticed few issues with capitalization on account settings modal, extra icons on `more...` links on the old sidebar and the abbreviations and general language improvements Nevy mentioned.
I will create a separate ticket so these translations/changes can be addressed by the contributors.
Can we capitalize the first letter of the language name on the dropdown in this PR? 
Screen Shot 2020-09-08 at 2 44 27 PM
Screen Shot 2020-09-08 at 3 30 26 PM

**Review by jgilliam17 — APPROVED** [review]
Thank you @metanerd 
- Verified Bulgarian language has been added in Account Settings>Display>Language, language title has been capitalized.
Follow up issue added for translation improvement (capitalization missing, abbreviations etc.) https://mattermost.atlassian.net/browse/MM-28522

**Review by ctlaltdieliet — COMMENTED** [review]
ok for me

**Review by ctlaltdieliet — COMMENTED** [review]
ok

**Pull request merged**